Formula/rio.rb

```diff
@@ -0,0 +1,28 @@
+class Rio < Formula
+  desc "A hardware-accelerated GPU terminal emulator powered by WebGPU, focusing to run in desktops and browsers."
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  version "0.0.7"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "pkg-config" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "rustup-init" => :test
+
+  on_linux do
+    depends_on "mesa"
+  end
+
+  def install
+    system "make build"
+    bin.install "target/release/rio"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/rio --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
added `assert_equal "0", shell_output("rio -e 'echo 1; exit' && echo $!").strip` let me know your thoughts @carlocab 🙏

Reply:
Doing
```ruby
system bin/"rio", "-e", "echo 1; exit"
```
should likely suffice for the test that you're trying to do. (`system` here will assert a non-failing exit status.)

Reply:
ah perfect, updated